src/components/input/select/index.tsx

```diff
@@ -155,8 +202,8 @@ export class SmoothlyInputSelect implements Input {
 	}
 	render() {
 		return (
-			<Host tabIndex={0} class={(this.missing ? "missing" : this.type) ?? ""}>
-				<main ref={element => (this.mainElement = element)}>{this.initialPrompt ?? "(none)"}</main>
+			<Host tabIndex={0} class={this.missing ? "missing" : ""}>
```

Comment:
`class={this.missing ? "missing" : ""}` can be written as `class={{missing: this.missing}}`